libkbfs/folder_branch_ops.go

```diff
@@ -2221,7 +2229,7 @@ func (fbo *folderBranchOps) createEntryLocked(
 	}
 
 	// verify we have permission to write
-	md, err := fbo.getMDForWriteLocked(ctx, lState)
+	md, err := fbo.getMDForWriteLockedForName(ctx, lState, name)
```

Comment:
This doesn't seem right.  If you did `touch /keybase/private/strib/a/b/c/name`, the error would be for `/keybase/private/strib/name`, right?  I think you need to include the full path using `dir`.